_posts/2017-11-08-cost-of-drupal-7.md

```diff
@@ -0,0 +1,180 @@
+
+#### Access to complementary tools
+
+The re-architecture towards object-orientation helped Drupal join the modern PHP community's framework ([Cake](https://cakephp.org/), [Phalcon](https://phalconphp.com/en/), [Zend](https://framework.zend.com/), [Slim](https://www.slimframework.com/), [Laravel](https://laravel.com/), [CodeIgniter](https://www.codeigniter.com/), [Symfony](https://symfony.com/), [Yii](http://www.yiiframework.com/), and [Fuel](https://fuelphp.com/) to name the most popular) development practices. One subtle, yet substantial value to that move is now many tools that are built to help support these other frameworks are available to Drupal as well. As the toolkit for modern web development grows richer and more robust, the more Drupal can utilize, the better. A couple examples we've recently used to help inform project quality and future maintenance costs are [Code Climate](https://codeclimate.com/) and [Scrutinizer](https://scrutinizer-ci.com/). These tools have little or no value analyzing a Drupal 6 or 7 site.
+
+## Our advice
+
+So, we've dumped a lot of information on you at this point, but it may still not be entirely clear what you should do with your outdated Drupal site. Ahead we provide general suggestions as well as what is pertinent for each version separately (Drupal 6 and 7) to site owners.
+
+### To everyone on Drupal <8
+
+We still have <3 for Drupal <8. It got us to where we are. But here's what we think you should consider about where you're going.
+
+1. **Plan 3 years out** if possible. A stitch, in time, saves nine, and every minute of planning saves 10 in execution. It's best to be in the driver's seat on these large investment decisions rather than at the mercy of fleeting security support. An experienced partner will help shepherd this process, but be vigilant regardless. Create a roadmap that accounts for substantial functional changes, a redesign, an integration to another system, or whatever is unique to your organization. This will help you use your resources most efficiently. It's understandably not as compelling to site owners and budget stakeholders to invest just for the sake of upgrading. The security implications will often be undervalued and overlooked. If your organization hasn't historically done this, it doesn't help you much in the present, but that's all the more reason to change your orientation now.
```

Comment:
This section is pretty vague to me. It sort of reads as businessy jargon...is there a way to be more specific about what site owners need to do?